contributors/devel/testing.md

```diff
@@ -225,3 +225,40 @@ version and the watch cache test is skipped.
 ## End-to-End tests
 
 Please refer to [End-to-End Testing in Kubernetes](e2e-tests.md).
+
+## Running your contribution in the Kubernetes CI
+Once you open a PR, prow will run pre-submit tests in CI.
+
+### Troubleshooting a failure
+Click on `Details` and look at the [`gubernator`](gubernator.k8s.io/) output for the test.
+
+#### Troubleshooting failures/flakes that are not caused by your change
+- Check if an issue has already been opened for that test failure/flake
+  - If yes, comment on it and link your PR, the failed run that affected you and any other information you think might be relevant
+  - If no, open a new issue and notify the appropriate SIG (see: SIG test escalation)
+
+#### SIG test escalation
+- Figure out corresponding sig from test name/description
+- Mention the sig's github handle on the issue, optionally cc the SIG's chair(s) (locate them under kubernetes/community/sig-<name>)
```

Comment:
some capitalizations:

`sig` -> `SIG`
`github` -> `GitHub`
`cc` -> `CC`

Reply:
Thanks, capitalized sig and github except where (I thought) it made sense to keep lower-case (e.g. in urls)

I changed cc to code format (`cc`) rather than capitalizing with the logic that it's sort of a command within a GitHub conversation... no strong opinions though. What do you think?

Reply:
sounds good :+1: